wbooks-api/app/serializers/book_serializer.rb

```diff
@@ -4,6 +4,7 @@ class BookSerializer < ActiveModel::Serializer
   attributes :id, :title, :author, :genre, :publisher, :year
 
   has_many :rents, key: :actual_rent do
-    object.rents.where('restitution >= ?', Date.current).first
+    current_date = Date.current
+    object.rents.where('loan <= ? AND restitution >= ?', current_date, current_date).first
```

Comment:
you could move this query to a rents scope
scope :actual, lambda {
  current_date = Date.current
  where('loan <= ? AND restitution >= ?', current_date, current_date).first
}

and you can use this in every rents collection

object.rents.actual

Reply:
Agreed, even if you don't do it for this PR it's good to know about scopes, they are useful when queries are used in multiple places